pkg/asset/ignition/bootstrap/bootstrap-in-place.go

```diff
@@ -7,11 +7,18 @@ import (
 	"github.com/openshift/installer/pkg/types"
 )
 
-// verifyBootstrapInPlace validate the number of control plane replica is one
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
 func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
 	if *installConfig.ControlPlane.Replicas != 1 {
 		return errors.Errorf("bootstrap in place require single control plane replica, current value: %d", *installConfig.ControlPlane.Replicas)
 	}
+	if installConfig.BootstrapInPlace == nil {
+		return errors.Errorf("Missing bootstrap in place configuration")
+	}
+	if installConfig.BootstrapInPlace.InstallationDisk == "" {
+		return errors.Errorf("bootstrap in place require installation disk")
```

Comment:
nit s/require/requires an/

Reply:
Going with `bootstrap in place requires installation configuration`